src/js/providers/video-listener-mixin.js

```diff
@@ -103,6 +103,9 @@ const VideoListenerMixin = {
     playing() {
         // When stalling, STATE_PLAYING is only set on timeupdate
         // because Safari and Firefox will fire "playing" before playback recovers from stalling
+        if (this.state === STATE_PLAYING) {
```

Comment:
This change will affect all providers and should not be made. Multiple  `PROVIDER_FIRST_FRAME` events are fine because these are internal events handled in the player with 'firstFrame' being the external deduped event.

If we're getting multiple playing events that is a problem with state change deduping or  `this.stallTime` not being treated properly in the shaka provider.

Reply:
>'firstFrame' being the external deduped event.

Ok, we can make it so that we only guard against firing PLAYING.

>If we're getting multiple playing events that is a problem with state change deduping or this.stallTime not being treated properly in the shaka provider.

The problem is that FF fires `timeupdate` before `playing`, causing the `PLAY` event to be fired twice.